support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/eventsModule.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { SvgTicket } from 'components/icons/ticket';
+
+import Text from 'components/text/text';
+import { LinkButton } from '@guardian/src-button';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { sendTrackingEventsOnClick } from 'helpers/productPrice/subscriptions';
+import { getMemLink, emailPreferences } from 'helpers/urls/externalLinks';
+
+const marginForButton = css`
+  margin-bottom: ${space[3]}px;
+`;
+
+
+const EventsModule = () => (
+  <div>
+    <SvgTicket />
+    <Text title="Guardian digital events">
+      <p>
+      Enjoy 6 free tickets to Guardian digital events in the first 3 months of your subscription.
+      You will be sent your unique redemption code by email shortly, enter your code 6 times at
+      the checkout for your chosen events to receive your free tickets.
```

Comment:
```suggestion
      Enjoy 6 free tickets to Guardian digital events in the first 3 months of your subscription.
      You will be sent your unique redemption code by email shortly, which you can use 6 times at
      the checkout for your chosen events to receive your free tickets.
```

Reply:
I think it's definitely an improvement @johnduffell but you would need to check this with Georgia Easteal as this text comes from her.